config/environments/test.rb

```diff
@@ -43,4 +43,6 @@
   config.middleware.use ClearanceBackdoor
 
   config.active_support.test_order = :random
+
+  config.cache_store = :dalli_store, 'localhost:11211'
```

Comment:
@maclover7 For some weird reason dalli + travis doesn't work if you pass in a connection string. Change this line to `config.cache_store = :dalli_store` and this will work.